src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php

```diff
@@ -296,30 +296,27 @@ abstract protected function createValidator(): ConstraintValidatorInterface;
 
 final class ConstraintViolationAssertion
 {
-    private ExecutionContextInterface $context;
-
     /**
      * @var ConstraintViolationAssertion[]
      */
     private array $assertions;
 
-    private string $message;
     private array $parameters = [];
     private mixed $invalidValue = 'InvalidValue';
     private string $propertyPath = 'property.path';
     private ?int $plural = null;
     private ?string $code = null;
-    private ?Constraint $constraint;
     private mixed $cause = null;
 
     /**
      * @internal
      */
-    public function __construct(ExecutionContextInterface $context, string $message, ?Constraint $constraint = null, array $assertions = [])
-    {
-        $this->context = $context;
-        $this->message = $message;
-        $this->constraint = $constraint;
+    public function __construct(
+        private ExecutionContextInterface $context,
+        private string $message,
+        private ?Constraint $constraint = null,
+        array $assertions = [],
```

Comment:
```suggestion
        private array $assertions = [],
```

Reply:
Not sure how this one wasn't detected, but here you go